src/file-uploader/file-uploader.js

```diff
@@ -52,6 +53,7 @@ function FileUploader(props: PropsT) {
     overrides.HiddenInput,
     StyledHiddenInput,
   );
+  const [Button, buttonProps] = getOverrides(overrides.Button, StyledButton);
```

Comment:
I'd like to maintain using the baseui button component here so that it maintains a consistent interface for devs looking to customize it. Check out this example of 'overriding': https://github.com/uber-web/baseui/blob/master/src/datepicker/datepicker.js#L141

Reply:
I believe this is what you meant. I tested it and it seems to work. Please tell me if there is something else you want me to do. Thanks!